text/0000-read-buf.md

```diff
@@ -252,6 +252,13 @@ impl Read for TcpStream {
 /// buffer that has been logically filled with data, a region that has been initialized at some point but not yet
 /// logically filled, and a region at the end that is fully uninitialized. The filled region is guaranteed to be a
 /// subset of the initialized region.
+///
+/// In summary, the contents of the buffer can be visualized as:
+/// ```not_rust
+/// [             capacity              ]
+/// [ filled |         unfilled         ]
+/// [    initialized    | uninitialized ]
```

Comment:
Is this possible?
```
buf: [ filled | unfilled | uninitialized ]
```

Reply:
What do you mean by that? The `unfilled` methods refer to the entire portion of the buffer that is not filled.

Reply:
Oh, it is a different concept, I forget that `uninitialized` means `unfilled` already.